src/client/components/header/Header.js

```diff
@@ -19,7 +24,21 @@ export const Header = (props) => {
       <div className="header-find-your-movie">
         {findYourMovie}
       </div>
-      <Search />
+      <SearchContainer match = {match}/>
     </div>
   );
-}
\ No newline at end of file
+}
+
+Header.propTypes = {
+  headerLabels: PropTypes.object,
```

Comment:
You can describe the shape of `headerLabel` by 
`headeLabel: PropTypes.shape({
  netflixRoulette: PropTypes.arrayOf(PropTypes.oneOf(['netflix', 'roulette'])),
 findYourMovie: PropTypes.string
})`

Reply:
Done)